roles/custom/matrix-prometheus-nginxlog-exporter/tasks/validate_config.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Fail if Prometheus not enabled
+  ansible.builtin.fail:
+    msg: >
+      You need to enable `matrix_prometheus_enabled` as this roles requires Prometheus for data storage.
+  when: "not matrix_prometheus_enabled"
```

Comment:
What if the user is using an external Prometheus server to scrape `/metrics/nginxlog`, etc.?

Perhaps we should remove this check, as it's not really a hard requirement.

Reply:
fixed in 3acc2b7

Reply:
but we can delete it all together. I prefer to leave it in, I believe quite a few will miss this otherwise and can't think of a downside for users who do.

Reply:
I'm not sure I understand why you'd like to keep this in.

The downside is for users who use an external Prometheus server (hosted elsewhere, not on the Matrix server) and want to crawl their `https://matrix.DOMAIN/metrics/nginxlog`, along with other metrics from `/metrics/*`.

I doubt people would enable nginxlog-exporter and forget to add `matrix_prometheus_enabled`. The worst that could happen is them saying "I don't see the metrics in my Prometheus.. oh wait, I don't have a Prometheus".

Reply:
> fixed in [3acc2b7](https://github.com/spantaleev/matrix-docker-ansible-deploy/commit/3acc2b7649b837a9d1babe30910c1ed834937dfe)

I guess you missed the commit 'above'.
The comment you reply should've been on this 'new commit' 3acc2b7. I think it's best to leave in with updated condition.
W/ the updated condition there's no downside.

Can remove if it's a hard 'no' for you.

Reply:
Oh.. I saw that but didn't quite understood it. Now that I look at it again, it seems fine!